config/validated_config_db_connector.py

```diff
@@ -22,6 +32,13 @@ def make_path_value_jsonpatch_compatible(table, key, value):
     return path, value
 
 def create_gcu_patch(op, table, key=None, value=None):
+    gcu_json_input = []
+    if op == "add" and not is_table_present_config_db(table):
```

Comment:
>is_table_present_config_db [](http://example.com/codeflow?start=27&length=26)

Need to check key or table existence.

Reply:
Also addressed this comment in PR https://github.com/sonic-net/sonic-utilities/pull/2452

Reply:
I see it resolved in 2452.